### ISSUE TITLE
Update Job Posts on company behalf

### ISSUE THREAD

**Issue by theGaryLarson** [Computing-For-All/nextjs-issue-tracker#39]
Initially this will be a function of admin but once it is bug-free and process is confirmed to work smoothly internally then we will open up this functionality Employer Dashboard.

**Comment by not-damien** [comment]
on admin branch

**Comment by theGaryLarson** [comment]
@not-damien Want to revisit this with the additional features requested by Bethany and Michael.

**Comment by theGaryLarson** [comment]
@not-damien  there isn't a way on UI/UX for deleting job posts as an admin yet. Think we should just add a delete button to the cards that's visible when logged in as an admin?

**Comment by not-damien** [comment]
Yeah we can do that, thats how it was working when the job post was a page before (It showed up if the user was an employer for the company, we can expand that to include the admin case )

**Comment by not-damien** [comment]
admin add and delete are fully functional now all that is needed is admin edit
